S2VX.Game/Story/Note/EditorHoldNote.cs

```diff
@@ -35,14 +35,14 @@ public class EditorHoldNote : HoldNote {
             var endFadeOut = EndTime + notes.FadeOutTime;
             var startTime = HitTime - notes.ShowTime;
 
+            UpdatePlacement();
```

Comment:
Is there a reason why we need to update the position of HoldNotes even when they're not visible?

Reply:
I moved it above the alpha check because not setting Size and Position caused the selecting whole screen bug, but I'll move it back and just set Size&Position before the alpha check since that's all we need.

Reply:
Hmm, I'm not sure if I like that. If in the future we have to create a new type of note then we'll have to remember to manage Size and Rotation separately from Position. Seems more intuitive for me to group them all (Position, Size, and Rotation) all together under Placement.

Reply:
True. Ok, I'll set Size&Rotation back in UpdatePlacement